src/winston-dynamodb.ts

```diff
@@ -0,0 +1,160 @@
+import * as winston from 'winston';
+import * as util from 'util';
+import * as AWS from 'aws-sdk';
+import * as uuid from 'node-uuid';
+import * as _ from 'lodash';
+import * as os from 'os';
+
+const hostname = os.hostname();
+
+function datify(timestamp) {
+  let dateTS = new Date(timestamp);
+  let date = {
+    year: dateTS.getFullYear(),
+    month: dateTS.getMonth() + 1,
+    day: dateTS.getDate(),
+    hour: dateTS.getHours(),
+    minute: dateTS.getMinutes(),
+    second: dateTS.getSeconds(),
+    millisecond: dateTS.getMilliseconds()
+  };
+
+  let keys = _.without(Object.keys(date), "year", "month", "day");
```

Comment:
NOTE; I'm not sure this was correct originally, so I tweaked it a bit to get it to compile.